### DISCUSSION TITLE
How can I add this specific parameter (path) through swagger configuration into Kong?

### DISCUSSION THREAD

**Discussion by joaocarvoli** [Kong/kong#8363] [Help]
Hi, if You already use swagger import into Kong, You know that Kong has a lot of parameters and I didn't find a specification teaching how someone would add them into swagger specification. My problem is this with the above specific parameter.

The print is about Konga...

![image](https://user-images.githubusercontent.com/62486484/151990673-118a75fe-a790-4e19-b723-cf6fa331607f.png)

Swagger specification bellow:

![image](https://user-images.githubusercontent.com/62486484/152018849-49f52175-2614-4fc3-b4c0-b9b87d460612.png)

I tried many possible ways that I know but I cannot add this through the swagger spec.

**Comment by hbagdi** [comment]
cc @gAmUssA
